.travis.yml

```diff
@@ -194,16 +194,27 @@ jobs:
     - <<: *create_manifest
       stage: manifest
       name: Create manifest for release
-      if: tag =~ $VERSION_TAG
+      if: tag =~ /^v.*/
       env:
         - IMAGE=$OAO_IMAGE_DOCKER_HUB
         - IMAGE=$OAO_IMAGE_RHCC
 
+      ######################### Prepare CSV #########################
+
+    - stage: release
+      name: Prepare CSV file
+      if: tag =~ /^v.*/
+      install: skip
+      workspaces:
```

Comment:
You don't really need the workspace, do you?

Reply:
You're right - removed